### PR TITLE
Adds Vox Sprites for all Remaining Clothing/Under/ + Ghost Uniform

### PR DESCRIPTION
**Pull request by KasparoVy**
Did this in a few sections, first I went through civvie jobs, then engineering, then medsci (as security was already done in another PR, that won't be covered here). Then after finding I had to make a certain change, I just added absolutely everything else. I'll break that into sections for the sake of pictures.

For Civilian jobs:
 - Sprites added: Clown, Mime, Hopwhimsy, Barber

voxciv

For Engineering jobs:
- Sprite added: Mechanic

voxmechanic

For Medsci jobs:
- Sprites added: Mortician's blackscrubs, Psychiatrist uniform, Psychiatrist turtleneck, all 'new' suffixed uniforms:

voxmedsci

**FOR EVERYTHING ELSE:**
Bathing suits/shorts:
voxswimnstrip

Everything in pants.dm (in the order in which they appear from top to bottom in code):
voxpants

Everything in miscellaneous.dm, syndicate.dm + ghost tourist uniform
voxmisc1

voxmisc2

voxmisc3

voxmisc4

Each section saw a conversion from a species_fit and a sprite_sheets for each obj to one at the top of the file under a common directory

### CODE REVIEW

File: code/modules/clothing/under/jobs/engineering.dm [hunk 1]
```diff
@@ -1,3 +1,10 @@
+//Fits all uniforms to Vox
+/obj/item/clothing/under/rank/
```
Comment (Fox-McCloud) [inline code/modules/clothing/under/jobs/engineering.dm]: this shouldn't be defined multiple times; likewise, making one for clothing/under will catch clothing/under/rank.

**Comment by Fox-McCloud** [comment]
If you're going to define one for clothing/under....then ANYTHING that's a child of clothing/under doesn't need species fit and species sprite sheets defined on it.

File: code/modules/clothing/under/jobs/engineering.dm (hunk 1 above)
Comment (KasparoVy) [inline code/modules/clothing/under/jobs/engineering.dm]: So what you're saying is if I defined obj/item/clothing/under/

then did the whole species_fit sprite_sheets business in another file, it'll apply here?

**Comment by KasparoVy** [comment]
Okay, so if I removed the /obj/item/clothing/under/rank with the Vox fit stuff and just left the one /obj/item/clothing/under/ in civilian.dm, it'll cover every other file in code/modules/clothing/under/jobs/?

**Comment by Fox-McCloud** [comment]
simply put if you define the fit and sprite list for clothing/under....ANYTHING that's a child of that will inherit those properties unless otherwise defined.

I'm not sure if there's any jumpsuits/pants/shorts/etc that aren't children of clothing/under.

**Comment by KasparoVy** [comment]
So even though I defined *obj/item/clothing/under/* down here in *code/modules/clothing/under/jobs/civilian.dm*, it'll affect, for example, the chameleon jumpsuit in code/modules/clothing/under/chameleon.dm because the chameleon jumpsuit is a child of */obj/item/clothing/under/*?

Sorry for seeming a bit dense, I'm just trying to make sure by changing it up I won't be unintentionally bork bork borking a bunch of other stuff by mistake.

**Comment by Fox-McCloud** [comment]
Yes. Ideally you should place it where clothing/under is first defined, though.

**Comment by KasparoVy** [comment]
Huh. Alright, since that's the case, I'll have to either exclude a bit of this PR or make it quite a bit bigger.

I'll work on the latter and see how it fares and if it becomes a problem I'll sort it the other way

**Comment by KasparoVy** [comment]
May as well slap a big ol' DNM on this because it'll take a little while to work the whole thing out. Wasn't expecting this, but I would've had to deal with it eventually anyway

**Comment by KasparoVy** [comment]
Just about to wrap this up. Few more sprites to go and I'll push a massive commit probably later tomorrow

**Comment by KasparoVy** [comment]
Running a bit behind schedule 'cause I was out most of the day doing last minute prep for classes starting back up on the 9th, 2 more sprites and I'll push the commit the evening of the 8th for sure.

**Comment by KasparoVy** [comment]
Done the sprites, time to get it working

**Comment by KasparoVy** [comment]
Alright, c'est fini.
Here comes commit. Then to test them all and post some pics.

File: code/modules/clothing/under/jobs/civilian.dm [hunk 2]
```diff
@@ -1,3 +1,10 @@
+//Fits all uniforms to Vox
+/obj/item/clothing/under/
```
Comment (Fox-McCloud) [inline code/modules/clothing/under/jobs/civilian.dm]: this isn't need anymore as you've already have this under clothing/under.

File: code/modules/clothing/under/jobs/engineering.dm (hunk 1 above)
Comment (Fox-McCloud) [inline code/modules/clothing/under/jobs/engineering.dm]: This still needs to go.

File: code/modules/clothing/under/jobs/medsci.dm [hunk 3]
```diff
@@ -1,3 +1,11 @@
+//Fits all uniforms to Vox
+/obj/item/clothing/under/rank/
```
Comment (Fox-McCloud) [inline code/modules/clothing/under/jobs/medsci.dm]: again, still needs to go.

**Comment by KasparoVy** [comment]
No problems so far, just checking for any last minute issues with the sprites at this point-- posting pictures as I go

**Comment by Fox-McCloud** [comment]
Don't forget to test jumpsuits on regular people to make sure they function properly as well =p

**Comment by KasparoVy** [comment]
Roger roger. There's a ton of sprites to go through, I might actually have to revisit testing tomorrow-- but aside from that one typo with the genetics_new jumpsuit, it's been problem free so far. But that just means it's easier for little bugs to hide

SOOON

**Comment by KasparoVy** [comment]
I'll pick testing back up tomorrow. Still loads to get through, but unless there's some weird typos I missed, I don't see there being a problem. Posted a few pictures of what I got through

**Comment by Fox-McCloud** [comment]
Just let us know when this is ready; hopefully if you complete testing soon we can get it in tonight.

**Comment by KasparoVy** [comment]
Alright, done. Couldn't find any more issues

**Comment by Fox-McCloud** [comment]
Heh, birdpanths.

**Pull request merged**